eduNEXT/seb-openedx
# Fg/seb-openedx blacklist and whitelist deprecation

### Deprecation of Fields in the SEB Plugin

### Description
In this PR, we have removed the `blacklist_chapters` and `whitelist_paths` fields from the `SebCourseConfiguration` model. The motivation behind this decision is that microfrontends are not designed to utilize these functionalities. To ensure better alignment and coherence with microfrontends and the current architecture, we have chosen to remove these features from the model.

### How to Test

- Ensure you have updated your database to the latest version reflecting the changes of this PR (migrate).
- Log in as an instructor user and create a course and assign SEB Configuration privileges in the admin site of LMS.
- Navigate through the different pages that compose the LMS course section and verify that they all work correctly.
- Ensure you test the views that previously relied on blacklist_chapters and whitelist_paths.
- Then, log out and log back in as a student user.
- Repeat the navigation process, ensuring all pages and functionalities are accessible and function correctly from the student's perspective.

### Additional Notes
With the removal of these fields from the model, it's essential that any custom configuration or adaptation that previously depended on these fields be reviewed and adjusted accordingly.

We appreciate any feedback or issue reporting related to this change.
